app-examples/src/main/java/net/spals/appbuilder/app/examples/grpc/rest/RestGrpcWebApp.java

```diff
@@ -0,0 +1,42 @@
+package net.spals.appbuilder.app.examples.grpc.rest;
+
+import net.spals.appbuilder.app.grpc.GrpcWebApp;
+import net.spals.appbuilder.config.service.ServiceScan;
+import net.spals.appbuilder.mapstore.core.MapStore;
+
+import java.io.IOException;
+
+/**
+ * A {@link GrpcWebApp} which includes RESTful APIs.
+ *
+ * @author tkral
+ */
+public class RestGrpcWebApp extends GrpcWebApp {
+
+    private static final String SERVICE_CONFIG_FILE_NAME = "config/rest-grpc-service.conf";
+    private static final int GRPC_PORT = 8080;
+
+    public static void main(final String[] args) throws IOException, InterruptedException {
+        final RestGrpcWebApp sampleGrpcWebApp = new RestGrpcWebApp();
+        sampleGrpcWebApp.start();
+        sampleGrpcWebApp.awaitTermination();
+    }
+
+    public RestGrpcWebApp() {
+        super(GRPC_PORT);
+    }
+
+    @Override
+    protected void configure(final Builder grpcWebAppBuilder) {
+        grpcWebAppBuilder
+            .enableCors()
+            .enableRestServer()
```

Comment:
With this line of configuration, you can run a REST server in parallel to your gRPC server.

gRPC uses Protocol Buffer serialization which makes it difficult to debug from the command line or from a browser. For example, you cannot use `curl` or `Postman`. By enabling a REST server here, AppBuilder will expose a second port on your gRPC server that accepts JSON over REST. This makes it easy to debug your gRPC endpoint using normal REST tools.